pyrovision/models/__init__.py

```diff
@@ -8,3 +8,4 @@
 from .densenet import *
 from .mobilenet import *
 from .ssresnet import *
+from .pyronear_model import *
```

Comment:
Since this isn't an architecture definition, I think there are two options:
- either we name it model_zoo and we will add other pretrained models in here whatever their type
- or, since it's a rexnet, we create a module rexnet.py and you add a rexnet1_0 function and define it like in you pyronear_model

I think the latter is better considering we could trained different rexnet, and for a given rexnet type (let's say rexnet1_0), if we train a better one, it will replace the previous.